File: decompiled/3.20.2/be/ehealth/businessconnector/mycarenet/memberdatav2/builders/ResponseObjectBuilder.java
package be.ehealth.businessconnector.mycarenet.memberdatav2.builders;

import be.ehealth.businessconnector.mycarenet.memberdatav2.domain.MemberDataBuilderResponse;
import be.ehealth.technicalconnector.exception.TechnicalConnectorException;
import be.fgov.ehealth.mycarenet.memberdata.protocol.v1.MemberDataConsultationResponse;

public interface ResponseObjectBuilder {
   MemberDataBuilderResponse handleConsultationResponse(MemberDataConsultationResponse var1) throws TechnicalConnectorException;
}

File: src/main/kotlin/org/taktik/connector/business/memberdatav2/builders/ResponseObjectBuilder.kt
package org.taktik.connector.business.memberdatav2.builders

import be.fgov.ehealth.mycarenet.memberdata.protocol.v1.MemberDataConsultationResponse
import org.taktik.connector.business.memberdatav2.domain.MemberDataBuilderResponse
import org.taktik.connector.technical.exception.TechnicalConnectorException
import org.taktik.connector.technical.service.etee.Crypto

interface ResponseObjectBuilder {
    @Throws(TechnicalConnectorException::class)
    fun handleConsultationResponse(consultResponse: MemberDataConsultationResponse, crypto: Crypto): MemberDataBuilderResponse?
}
